src/node_jsvmapi.h

```diff
@@ -116,80 +116,99 @@ enum napi_valuetype {
   napi_function,
 };
 
-// Environment
-NODE_EXTERN napi_env napi_get_current_env();
+enum napi_errorvalue {
```

Comment:
I think everyone liked the idea of using some bits to indicate status like this but we didn't settle on if we should actually do it or if it was practical or not. Hence, I haven't done it here. I suppose we could use a macro like NAPI_EPILOGUE to always check the pending exception support?